packages/portal/spec-renderer/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kong-ui/portal-spec-renderer",
-  "version": "0.1.1",
+  "version": "0.1.2",
```

Comment:
@kaiarrowood did you bump the version?

Publishing is automated in this repository, meaning you should never need to manually touch the version or publish.